demisto_sdk/commands/validate/file_validator.py

```diff
@@ -301,15 +302,15 @@ def validate_modified_files(self, modified_files):  # noqa: C901
                 if not self.id_set_validator.is_file_valid_in_set(file_path):
                     self._is_valid = False
 
-            elif checked_type(file_path, YML_INTEGRATION_REGEXES):
+            elif checked_type(file_path, YML_INTEGRATION_REGEXES) or file_type == 'integration':
```

Comment:
What cases are you catching here (and elsewhere that you added this logic) that weren't being caught before?

Reply:
The regexes are broken this is another way of supporting it, it's done in other methods as well until we fix the regexes.

Reply:
tov